package.json

```diff
@@ -18,6 +18,7 @@
     "test": "ng test --watch=false  --browsers=ChromeHeadless --code-coverage=true",
     "watch": "ng test",
     "lint": "ng lint",
+    "lintfix": "ng lint --fix",
```

Comment:
Lets also add this to the package.json build (so that auto fixing of lint issues happens as part of the build), for Application Insights this happens via the grunt configuration.

Reply:
So it would become
```json
"build": "npm run lintfix && ng build --configuration production",
```